WhisperAPI/WhisperAPI/Models/ConversationContext.cs

```diff
@@ -6,19 +6,26 @@ namespace WhisperAPI.Models
 {
     public class ConversationContext
     {
-        public ConversationContext(Guid chatkey, DateTime datetime)
+        public ConversationContext(Guid chatkey, DateTime datetime) 
+            : this()
         {
             this.ChatKey = chatkey;
             this.StartDate = datetime;
         }
 
-        private ConversationContext()
+        public ConversationContext()
         {
+            this.SearchQuerries = new List<SearchQuerry>();
+            this.SuggestedDocuments = new List<SuggestedDocument>();
         }
 
         [Key]
         public Guid ChatKey { get; set; }
 
         public DateTime StartDate { get; set; }
+
+        public List<SearchQuerry> SearchQuerries { get; set; }
+
+        public List<SuggestedDocument> SuggestedDocuments { get; set; }
```

Comment:
This should be a data structure that prevent duplicates. I think it is a Set or HashSet. Add a unit test to see that it works by adding two documents that are different objects, but have same values (title and Uri).